### ISSUE TITLE
Labware Library: Backend: Load new labware defs into legacy Placeable

### ISSUE THREAD

**Issue by sfoster1** [Opentrons/opentrons#3145]
As an Opentrons user, when I see a piece of labware in the labware library I want to immediately be able to add it to my protocol on APIv1.

As an Opentrons user, I don't want to have multiple definitions or multiple names for the same pieces of labware.

As an opentrons developer and maintainer, I don't want to ever touch legacy labware defs again.

To implement all this, add capability to the robot singleton to load new labware definitions into Placeable. This unblocks the labware library by setting a single goal: load all new labware defs, use their new names, all good.

Done is:

- `containers.load_labware` in the legacy API will load
  - standard APIv1 labware
  - previously-created APIv1 custom labware
  - standard APIv2 labware
- JSON protocols when using APIv1 will load
  - standard APIv1 labware by name
  - standard APIv2 labware by name

Out of scope: loading APIv2 labware definitions from JSON protocol files

**Comment by Laura-Danielle** [comment]
Closed with #3309 and #3435
